0-ad-history-mod-package/simulation/templates/structures/spart/barracks.xml

```diff
@@ -10,6 +10,16 @@
     <SpecificName>Stratopedon</SpecificName>
+    <History>
+        The Agoge, the Spartan training and education system, was introduced somewhere around the 8th century BC by Lycurgus, the legendary lawgiver of Sparta. It was a state-sponsored program to teach children the values and skills necessary to fight for their homeland.Of course, this system was exclusive to the children of Spartan citizens, the highest class of Spartan society, the Spartiates. Although the Agoge was primarily focused on training boys into soldiers, the girls were also required to do different forms of exercise to stay fit, as it was believed that only men and women in peak physical conditions would produce strong and healthy offspring.
+        Boys entered the Agoge probably around the age of seven, leaving their families to live in communal barracks with other boys. They received a basic education in literature, music, and history. The group was supervised by an elderly Spartan veteran, who acted as their mentor and instructor, and by a group of young men aged 19 to 20, who enforced discipline and rules. Not only them, but all Spartan citizens were given the power to punish the boys by any means, be it the whip or the stick, for misbehavior or disobedience. Through this harsh regime, the boys learned to respect their elders, to be resourceful, to endure pain and hardship, and to obey orders without question. Spartans were renowned for their discipline and self-restraint, avoiding eye contact or conversation with strangers in public.
+        As a preparation for times of war and crisis, the boys were forced to undergo harsh labor and physical training, with minimal clothing and no footwear whatsoever, no matter the weather or terrain (walking barefoot was believed to strengthen the feet). They learned to control their emotions, resist physical pain, and detach from any personal possession.
+        Competition also played a very important role in the Agoge. The boys competed in various activities such as ball games, wrestling, races, and hunting meant to improve their agility, concentration, coordination, and teamwork skills. The games were often deliberately designed to create rivalries between groups or individuals that strived to outperform each other not only in training but also on the battlefield.
+        As the boys grew older, they had to spend a certain period of time alone in the wilderness, known as "fox time". This was meant to test everything they had learned so far. In addition to hunting for food, it was encouraged to steal the food necessary to survive. However, when caught, the boys were punished severely and beaten up, not for the act of stealing itself but rather for being careless and caught. There is a tale of a Spartan boy who had stolen a fox and hidden it under his cloak. The fox started biting the boy, but instead of taking the embarrassment and shame of being caught stealing, he endured the pain and let the fox bite and gnaw him to death.
```

Comment:
```suggestion
        As the boys grew older, they had to spend a certain period of time alone in the wilderness, known as "fox time". This was meant to test everything they had learned so far. In addition to hunting for food, it was encouraged to steal the food necessary to survive. However, when caught, the boys were punished severely and beaten up, not for the act of stealing itself but rather their carelessness of being caught. There is a tale of a Spartan boy who had stolen a fox and hidden it under his cloak. The fox started biting the boy, but instead of taking the embarrassment and shame of being caught stealing, he endured the pain and let the fox bite him to death.
```